.pre-commit-config.yaml

```diff
@@ -394,12 +401,6 @@ repos:
         pass_filenames: false
         require_serial: true
         additional_dependencies: ['pyyaml']
-      - id: pre-commit-descriptions
-        name: Check if pre-commits are described
-        entry: ./scripts/ci/pre_commit/pre_commit_check_pre_commits.sh
-        language: system
-        files: ^.pre-commit-config.yaml$|^STATIC_CODE_CHECKS.rst|^breeze-complete$
-        require_serial: true
```

Comment:
Removed this as it was duplicated:
https://github.com/apache/airflow/blob/3ddf4d0c4501530f31f6dadb69fcb461acf24387/.pre-commit-config.yaml#L397-L402
https://github.com/apache/airflow/blob/3ddf4d0c4501530f31f6dadb69fcb461acf24387/.pre-commit-config.yaml#L459-L464